### PR TITLE
Update APF metrics again

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
This PR updates the user-facing documentation for API Priority and Fairness to include some recently introduced Prometheus metrics.

### CODE REVIEW

**Comment by cyang49** [comment]
@MikeSpreitzer Just a reminder, the documentation is still out-of-date `{{< feature-state state="beta"  for_k8s_version="v1.20" >}}`. I checked the [v1.20 metrics.go](https://github.com/kubernetes/apiserver/blob/release-1.20/pkg/util/flowcontrol/metrics/metrics.go), and it doesn't have the metrics you added documentation for yet.

**Comment by MikeSpreitzer** [comment]
@cyang49: there is some confusion here.  You cited the tag at the head of the document describing the maturity of the feature.  That is an accurate description.

This PR is updating the `main` branch of the website to make it consistent with the `master` branch of the code.  Additional work may then be needed to update release-specific branch(es).

**Comment by cyang49** [comment]
> @cyang49: there is some confusion here. You cited the tag at the head of the document describing the maturity of the feature. That is an accurate description.
Ah, thanks for the clarification. I thought that the tag means the documentation is targeting release 1.20

**Comment by lavalamp** [comment]
Tech LGTM

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -502,18 +506,47 @@ poorly-behaved workloads that may be harming system health.
   values `waiting` and `executing`) and `priority_level`.  Each
   histogram gets observations taken periodically, up through the last
   activity of the relevant sort.  The observations are made at a high
-  rate.
+  rate.  Each observed value is a ratio, between 0 and 1, of a number
```
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/flow-control.md:509]: Typography nit throughout: only one space following periods
```suggestion
  rate. Each observed value is a ratio, between 0 and 1, of a number
```

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 2]
```diff
@@ -502,18 +506,47 @@ poorly-behaved workloads that may be harming system health.
...
   water marks show the range of values that occurred between samples.
 
+* `apiserver_flowcontrol_priority_level_seat_count_samples` is a
+  histogram vector of observations of the utilization of a priority
+  level's concurrency limit, broken down by `priority_level`.  This
+  utilization is the fraction (number of seats occupied) /
+  (concurrency limit).  This metric considers all stages of execution
+  (both normal and the extra delay at the end of a write to cover for
+  the corresponding notification work) of all requests except WATCHes;
+  for those it considers only the initial stage that delivers
```
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/flow-control.md:532]:
```suggestion
  for those, it considers only the initial stage that delivers
```

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 3]
```diff
@@ -502,18 +506,47 @@ poorly-behaved workloads that may be harming system health.
...
+  notifications of pre-existing objects.  Each histogram in the vector
+  is also labeled with `phase: executing` (there is no seat limit for
+  the waiting phase).  Each histogram gets observations taken
```
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/flow-control.md:535]: Use [active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)
```suggestion
  the waiting phase).  Each histogram shows observations
```

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 4]
```diff
@@ -502,18 +506,47 @@ poorly-behaved workloads that may be harming system health.
...
+  periodically, up through the last activity of the relevant sort.
+  The observations
+  are made at a high rate.  
+
+* `apiserver_flowcontrol_priority_level_seat_count_watermarks` is a
+  histogram vector of high or low water marks of the utilization of a
```
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/flow-control.md:541]: Use [simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice); an idiom like "water mark" has multiple meanings in English and is difficult to localize. Might "usage mark" be equally accurate?
```suggestion
  histogram vector of high or low marks of the utilization of a
```

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 5]
```diff
@@ -502,18 +506,47 @@ poorly-behaved workloads that may be harming system health.
...
+  priority level's concurrency limit, broken down by `priority_level`
+  and `mark` (which takes on values `high` and `low`).  Each histogram
+  in the vector is also labeled with `phase: executing` (there is no
+  seat limit for the waiting phase).  The water marks are accumulated
```
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/flow-control.md:545]: See the preceding note about "water marks"
```suggestion
  seat limit for the waiting phase).  The usage marks are accumulated
```

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 6]
```diff
@@ -502,18 +506,47 @@ poorly-behaved workloads that may be harming system health.
...
+  over windows bounded by the times when an observation was added to
+  `apiserver_flowcontrol_priority_level_seat_count_samples`.  These
+  water marks show the range of values that occurred between samples.
```
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/flow-control.md:548]: See the preceding note about "water marks"
```suggestion
  usage marks show the range of values that occurred between samples.
```

**Review by zacharysarah — APPROVED** [review]
@MikeSpreitzer Thanks for your patience. This is solid. ✨ 

All of this feedback is non-blocking, but see the note about "water marks" being idiomatic and difficult to localize. Also for future docs, please use only one space after periods in sentences.

**Comment by zacharysarah** [comment]
@MikeSpreitzer I see a LGTM from @lavalamp, so I'll approve this PR with a non-blocking hold to give you the chance to see feedback prior to merge. When you're ready, feel free to `/hold cancel`.

/lgtm
/approve
/hold

**Comment by reylejano** [comment]
Going to unhold and let this PR merge and I will open a follow-up PR to address @zacharysarah 's suggestions
/unhold

**Pull request merged**